homeassistant/components/withings/__init__.py

```diff
@@ -56,7 +53,7 @@
                     vol.Optional(CONF_CLIENT_SECRET): vol.All(
                         cv.string, vol.Length(min=1)
                     ),
-                    vol.Optional(const.CONF_USE_WEBHOOK, default=False): cv.boolean,
+                    vol.Optional(const.CONF_USE_WEBHOOK): cv.boolean,
```

Comment:
I think you want to deprecate this?

Reply:
For follow-up PR maybe with this and other deprecated raise an issue so we can remove it later on?

Reply:
Yes, for a follow-up